doc/reference/networking/lwm2m.rst

```diff
@@ -254,14 +254,26 @@ To use the LwM2M library, start by creating an LwM2M client context
 	/* LwM2M client context */
 	static struct lwm2m_ctx client;
 
-Create callback functions for LwM2M resources that you wish to have actions
-for:
+Create callback functions for LwM2M resources that you wish to have create or
+delete actions for:
 
 .. code-block:: c
 
-	static int device_reboot_cb(uint16_t obj_inst_id)
+	static int resource_create_cb(uint16_t obj_inst_id)
 	{
-		LOG_INF("Device rebooting.");
+		LOG_INF("Resource created.");
+		return 0; /* wont reach this */
+	}
+
+Create callback functions for LwM2M resource exuctions:
+
+.. code-block:: c
+
+	static int device_reboot_cb(uint16_t obj_inst_id,
+                              uint8_t *args, uint16_t args_len)
+	{
+		LOG_INF("Device rebooting. (Argument list (%u bytes): %s)",
+            args_len, log_strdup(args));
```

Comment:
I don't think this will actually work, note that `args` is not a C-string (it's not NULL terminated). And `args` can be NULL pointer as well.

Reply:
I will remove it. I expected it to be a C-string as the arguments are in ANBF format.